__tests__/LoginAvatar-test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {LoginAvatar} from '../src/LoginAvatar';
+import { mountWithIntl, shallowWithIntl } from '../src/helpers/intl-enzyme-test-helper.js';
+
+function setup() {
+  const props = {
+    isAuthenticated: true,
+    dispatch: jest.fn(),
+    intl: {
```

Comment:
Probably makes sense to factor out the intl specific props into a shared module?  This is going to be a lot of copy & paste over time, and if react-intl changes, we'll end up with a mess.